ci/ctest_to_gitlab.sh

```diff
@@ -67,7 +67,7 @@ JOB_TEMPLATE="
   variables:
     SLURM_CPUS_PER_TASK: {{CPUS_PER_TASK}}
     SLURM_NTASKS: {{NTASKS}}
-    SLURM_TIMELIMIT: '10:00'
+    SLURM_TIMELIMIT: '20:00'
```

Comment:
codecov runs got extra time because of report generation that was very slow (with lcov, now with fastcov is better).
Therefore, even keeping the extra 5 min for report generation it would be 15 min.
```suggestion
    SLURM_TIMELIMIT: '15:00'
```

Reply:
Updated.